Patch by Antonio Galea <ant9000@netwise.it> to solve the socket number overflow
from #345331

--- a/netstat.c
+++ b/netstat.c
@@ -317,8 +317,8 @@
 
         strncpy(inode_str, lname+PRG_SOCKET_PFXl, inode_str_len);
         inode_str[inode_str_len] = '\0';
-        *inode_p = strtol(inode_str,&serr,0);
-        if (!serr || *serr || *inode_p < 0 || *inode_p >= INT_MAX) 
+        *inode_p = strtoul(inode_str,&serr,0);
+        if (!serr || *serr)
             return(-1);
     }
     return(0);
@@ -338,8 +338,8 @@
     {
         char *serr;
 
-        *inode_p=strtol(lname + PRG_SOCKET_PFX2l,&serr,0);
-        if (!serr || *serr || *inode_p < 0 || *inode_p >= INT_MAX) 
+        *inode_p=strtoul(lname + PRG_SOCKET_PFX2l,&serr,0);
+        if (!serr || *serr)
             return(-1);
     }
     return(0);
